### PR TITLE
Fix bug for content deletion reported resources and files

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Fixes regression introduced by https://github.com/learningequality/kolibri/pull/8506 due to the collapsing of resource counting for full and partial channel deletion
* Simplifies how the resources and files being deleted are counted

## Reviewer guidance
Partially import content for a channel
Delete all content from the channel
Confirm that the reported deletion matches the number of imported resources and files, and not the whole channel.

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by radinamatic** [comment]
Number of deleted resources is looking correct! 💯 


https://user-images.githubusercontent.com/1457929/164109724-7b2533e5-9561-4362-b6e2-75a2c5e83368.mp4

**Review by radinamatic — APPROVED** [review]
🎉 :shipit:

**Review by jredrejo — APPROVED** [review]
Honestly, I fully understand this new code but the old `get_import_export_data` ... is quite  a mess where it's hard to follow the data flow.  

Happy to see this clean implementation and I'd love to see that function replaced by a similiar approach in the other places where it's used (`exportcontent.py` , `importcontent.py `& `plugins/device/api.py` )
